src/Grapevine.Tests/Server/PublicFolderFacts.cs

```diff
@@ -190,124 +267,325 @@ public void TrimsLeadingAndTrailingWhitespace()
             }
         }
 
-        public class RespondWithFileMethod
+        public class CreateDirectoryListKeyMethod
+        {
+            [Fact]
+            public void ReplacesBackslashWithForwardslash()
+            {
+                var folder = new PublicFolder();
+                var path = Path.Combine(folder.FolderPath, "path1", "path2");
+
+                path.Contains(@"\").ShouldBeTrue();
+                path.Contains(@"/").ShouldBeFalse();
+
+                path = folder.GetDirectoryListKey(path);
+
```

Comment:
I think what I was trying to get at was that I couldn't really tell that was what it does from the name, maybe a different name like GetRoutePath() or something. Unless it is doing something else that I'm not seeing.